proposals/mobile-document-request-api-proposal.md

```diff
@@ -1,6 +1,6 @@
 # ⚠️ This proposal has been superseded ⚠️
 
-- Please refer to the [Digital identities spec](https://wicg.github.io/digital-identities/) instead. It represents the community's consensus.
+- Please refer to the [Digital Credentials API spec](https://www.w3.org/TR/digital-credentials/) instead. It represents the community's consensus.
```

Comment:
```suggestion
- Please refer to the [Digital Credentials spec](https://www.w3.org/TR/digital-credentials/) instead. It represents the community's consensus.
```

Reply:
Yeah, we should fix that too... it's not wrong though, it does define an API... but also the general model and concepts for "digital credentials". 

Let me file a bug to revise that a bit. That's all part of the spec's evolution (we will rewrite the introduction and abstract problems 100 times)